components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -48,12 +48,15 @@ function getTopContributorLanguages(contributor: DbUser) {
   // some contributors will have empty language objects so we will pull their popular language from the interests field instead of defaulting to nothing
   const entries = Object.entries<string>(contributor.languages);
   if (entries.length === 0) {
-    return [contributor.interests];
+    return contributor.interests.split(",").slice(0, 2);
   }
-  return entries
-    .sort(([, a], [, b]) => (a < b ? -1 : 1))
-    .slice(0, 2)
-    .map(([language]) => language);
+
+  return entries.length === 0
+    ? contributor.interests.split(",").slice(0, 2)
+    : entries
```

Comment:
The split on interests is not necessary.

```suggestion
  return (entries.length === 0
    ? contributor.interests.split(",")
    : entries)
```